### DISCUSSION TITLE
[Feature Request]: Arm Ampere support for image builder and servers?

### DISCUSSION THREAD

**Discussion by mannp** [mysticaltech/terraform-hcloud-kube-hetzner#715] [Ideas]
The new Ampere servers seem to be a decent price for the number of cores and memory.

Would be a welcome addition :)

Cheers

**Comment by gefloh** [comment]
Yes, the price seems great, would love to try these.

**Comment by mysticaltech** [comment]
Yep, now supported, since a long time ago.
